src/client/modules/my/graphExample/graphExample.ts

```diff
@@ -0,0 +1,76 @@
+import { LightningElement } from 'lwc';
+
+function getData() {
+    /*
+    return fetch('/api/v1/gameLogs')
+        .then(response => response.json());
+    */
+    return Promise.resolve([{
+        id: 0,
+        game_label: "20210521a",
+        player_num: 5,
+        player_name: "Matt",
+        victory_points: 14,
+        place: 3
+    }, {
+        id: 1,
+        game_label: "20210521b",
+        player_num: 2,
+        player_name: "Matt",
+        victory_points: 38,
+        place: 2
+    }, {
+        id: 2,
+        game_label: "20210521c",
+        player_num: 3,
+        player_name: "Matt",
+        victory_points: 22,
+        place: 1
+    }, {
+        id: 3,
+        game_label: "20210521d",
+        player_num: 2,
+        player_name: "Matt",
+        victory_points: 30,
+        place: 2
+    }]);
+}
+
+const placeNumberToString = ['Zeroth', 'First', 'Second', 'Third', 'Fourth', 'Fifth', 'Sixth'];   // lazy way to convert; fine given we have a very small list
+
+export default class GraphExample extends LightningElement {
+    victoryPointTrendData = [];
+    placeFrequencyData = [];
+    hasRendered = false;
+
+    async renderedCallback() {
+        // Avoids some infinite loop of trying to re-render. No idea why
+        if (!this.hasRendered) {
+            this.hasRendered = true;
+            const data = (await getData()).filter((d) => d.player_name === "Matt");
```

Comment:
Once Eli's PR gets merged, this should fairly easily be able to pull real data, and be re-worked for the different charts we want to show (we'll have to figure out what we even want to show)